container/quote-server/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN echo "deb [trusted=yes arch=amd64] file:/sgx_debian_local_repo jammy main" |
 RUN apt-get update && apt install -y libtdx-attest libtdx-attest-dev
 
 COPY service /service
```

Comment:
minor suggestion, can we just copy the service/quote-server folder to the container? like 'COPY service/quote-server service /service'